spec/libraries/azure_engine_spec.rb

```diff
@@ -115,24 +115,22 @@ def sign
     end
 
     it 'succeeds to send message' do
-      expect(AzureEngine.send_message('hoge')).to eq(nil)
+      expect(SlideHub::Cloud::Engine::Azure.send_message('hoge')).to eq(nil)
```

Comment:
__[reek]__
* [UnusedParameters] has unused parameter 'container' [:ref](https://github.com/troessner/reek/blob/master/docs/Unused-Parameters.md)
* [UnusedParameters] has unused parameter 'options' [:ref](https://github.com/troessner/reek/blob/master/docs/Unused-Parameters.md)
* [UtilityFunction] doesn't depend on instance state (maybe move it to another class?) [:ref](https://github.com/troessner/reek/blob/master/docs/Utility-Function.md)